src/main/java/io/neonbee/job/JobVerticle.java

```diff
@@ -74,12 +74,20 @@ public String getName() {
     @Override
     public void start() {
         if (!NeonBee.get(getVertx()).getOptions().shouldDisableJobScheduling()) {
+            if (schedule.isPeriodic() && isTemporalAdjusterInvalid()) {
+                throw new IllegalStateException("JobVerticle was configured with an invalid JobSchedule");
+            }
             scheduleJob();
         } else {
             finalizeJob();
         }
     }
 
+    private boolean isTemporalAdjusterInvalid() {
+        Instant testInstant = now();
+        return testInstant.with(schedule).compareTo(testInstant) <= 0;
```

Comment:
Mabe we should even call it multiple times, to see if it adjuste forward every time, just to be sure.

Reply:
done